Signal/src/call/CallService.swift

```diff
@@ -1198,6 +1198,7 @@ protocol CallServiceObserver: class {
             call!.hasLocalVideo)
     }
 
+    //TODO only fire this when it's changed? as of right now it gets called whenever you e.g. lock the phone while it's incoming ringing.
```

Comment:
Yes, we over-fire this.  We could certainly use a local property to de-duplicate these messages but I haven't bothered yet.